cmd/create.go

```diff
@@ -142,9 +137,16 @@ A full list of component types that can be deployed is available using: 'odo com
 			fmt.Printf("Please wait, creating %s component ...\n", componentName)
 			err = component.Build(client, componentName, applicationName, false, true, stdout)
 			checkError(err, "")
-			fmt.Printf("Component '%s' was created.\n", componentName)
-			fmt.Printf("To push source code to the component run 'odo push'\n")
 		}
+		ports, err := component.GetComponentPorts(client, componentName, applicationName)
+		checkError(err, "")
+		fmt.Printf("Component '%s' was created", componentName)
+		if len(ports) > 1 {
+			fmt.Printf(" and ports %s were opened\n", strings.Join(ports, ","))
+		} else if len(ports) == 1 {
+			fmt.Printf(" and port %s was opened\n", ports[0])
+		}
+		fmt.Printf("To push source code to the component run 'odo push'\n")
```

Comment:
The code above is a bit too cluttered. Please add newlines / spacing between the different parts of the code. For example:

```go
		ports, err := component.GetComponentPorts(client, componentName, applicationName)
		checkError(err, "")
		fmt.Printf("Component '%s' was created", componentName)

		if len(ports) > 1 {
			fmt.Printf(" and ports %s were opened\n", strings.Join(ports, ","))
		} else if len(ports) == 1 {
			fmt.Printf(" and port %s was opened\n", ports[0])
		}

		fmt.Printf("To push source code to the component run 'odo push'\n"
```

Reply:
This helps ensure clarity in the code when reading

Reply:
Ack